*** tests/A-01-09-M-ok.out	2022-10-10 11:47:13.000000000 +0100
--- tests/A-01-09-M-ok.outhyp	2022-10-24 12:45:37.573220375 +0100
***************
*** 23,29 ****
  6 - Mostrar saldo de cliente
  0 - Sair
  Escolha uma opção: CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|0|0|0
- CLIENT|cli002|Nome Cliente|12345|NORMAL|YES|0|0|0
  CLIENT|cli003|Outro Nome|54321|NORMAL|YES|0|0|0
  GESTÃO DE CLIENTES
  1 - Visualizar cliente
--- 23,28 ----
