*** tests/A-07-01-M-ok.out	2022-10-02 13:23:26.000000000 +0100
--- tests/A-07-01-M-ok.outhyp	2022-10-20 13:46:51.413979226 +0100
***************
*** 32,38 ****
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: BASIC|100000|cli002|IDLE|0|0
  GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
--- 32,38 ----
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: BASIC|100000|cli002|ON|0|0
  GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
