*** tests/A-08-06-M-ok.out	2022-10-05 21:24:49.000000000 +0100
--- tests/A-08-06-M-ok.outhyp	2022-10-20 14:15:30.649764698 +0100
***************
*** 16,25 ****
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: BASIC|969001|cli001|IDLE|0|0
! BASIC|969002|cli002|IDLE|0|0
! BASIC|969003|cli003|IDLE|0|0
! BASIC|999000|cli001|IDLE|0|0
  GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
--- 16,22 ----
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: BASIC|999000|cli001|IDLE|0|0
  GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
