*** testsBoth/A-12-04-M-ok.out	2022-10-30 22:57:52.000000000 +0000
--- testsBoth/A-12-04-M-ok.outhyp	2022-11-03 11:23:42.043862986 +0000
***************
*** 29,35 ****
  Escolha uma opção: 9
  Número do terminal: 969006
  Tipo de comunicação a iniciar (VOICE, VIDEO): VOICE
! O terminal de destino ('969006') está desligado.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 29,35 ----
  Escolha uma opção: 9
  Número do terminal: 969006
  Tipo de comunicação a iniciar (VOICE, VIDEO): VOICE
! O terminal '969001' não pode iniciar comunicações BASIC
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
