*** testsBoth/A-12-03-M-ok.out	2022-11-03 11:42:49.949159519 +0000
--- testsBoth/A-12-03-M-ok.outhyp	2022-11-03 13:55:08.744655274 +0000
***************
*** 62,68 ****
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 11
! VOICE|1|969004|969001|0|0|ONGOING
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 62,68 ----
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 11
! VOICE|1|969001|969004|0|0|ONGOING
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
