*** testsBoth/A-12-10-M-ok.out	2022-11-03 11:42:49.953159519 +0000
--- testsBoth/A-12-10-M-ok.outhyp	2022-11-03 13:55:09.316659685 +0000
***************
*** 41,47 ****
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 11
! VIDEO|1|969011|969012|0|0|ONGOING
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 41,47 ----
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 11
! VIDEO|1|969012|969011|0|0|ONGOING
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
