*** testsBoth/A-13-03-M-ok.out	2022-10-30 22:57:53.000000000 +0000
--- testsBoth/A-13-03-M-ok.outhyp	2022-11-03 11:23:43.608075318 +0000
***************
*** 41,47 ****
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 11
! VIDEO|1|969015|969014|0|0|ONGOING
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 41,47 ----
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 11
! VIDEO|1|969014|969015|0|0|ONGOING
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
