*** testsBoth/A-15-05-M-ok.out	2022-11-03 11:42:49.953159519 +0000
--- testsBoth/A-15-05-M-ok.outhyp	2022-11-03 13:57:40.533825870 +0000
***************
*** 42,48 ****
  0 - Sair
  Escolha uma opção: 10
  Duração da comunicação (em minutos): 30
! Custo da comunicação: 900 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 42,48 ----
  0 - Sair
  Escolha uma opção: 10
  Duração da comunicação (em minutos): 30
! Custo da comunicação: 0 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 81,87 ****
  7 - Mostrar terminais com saldo positivo
  0 - Sair
  Escolha uma opção: 1
! VIDEO|1|969011|969014|30|900|FINISHED
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
--- 81,87 ----
  7 - Mostrar terminais com saldo positivo
  0 - Sair
  Escolha uma opção: 1
! VIDEO|1|969011|969014|30|0|FINISHED
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
