*** testsBoth/A-16-02-M-ok.out	2022-10-30 22:57:55.000000000 +0000
--- testsBoth/A-16-02-M-ok.outhyp	2022-11-03 11:23:45.160285442 +0000
***************
*** 22,27 ****
--- 22,29 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
+ 8 - Enviar comunicação de texto
+ 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 4
***************
*** 34,39 ****
--- 36,43 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
+ 8 - Enviar comunicação de texto
+ 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 4
***************
*** 46,51 ****
--- 50,57 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
+ 8 - Enviar comunicação de texto
+ 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 0
