*** testsBoth/A-18-09-M-ok.out	2022-10-30 22:57:56.000000000 +0000
--- testsBoth/A-18-09-M-ok.outhyp	2022-11-03 17:20:52.218779747 +0000
***************
*** 159,165 ****
  Escolha uma opção: 1
  Identificador do cliente: cli001
  CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|4|0|0
! S2I|969008
  GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
--- 159,165 ----
  Escolha uma opção: 1
  Identificador do cliente: cli001
  CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|4|0|0
! S2I|969001
  GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
