*** testsBoth/A-19-07-M-ok.out	2022-10-30 22:57:58.000000000 +0000
--- testsBoth/A-19-07-M-ok.outhyp	2022-11-03 13:44:12.351189963 +0000
***************
*** 37,42 ****
--- 37,44 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
+ 8 - Enviar comunicação de texto
+ 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 3
***************
*** 49,54 ****
--- 51,58 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
+ 8 - Enviar comunicação de texto
+ 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 0
***************
*** 67,72 ****
--- 71,78 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
+ 8 - Enviar comunicação de texto
+ 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 3
***************
*** 79,84 ****
--- 85,92 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
+ 8 - Enviar comunicação de texto
+ 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 0
